app/notes/CreateNote.tsx

```diff
@@ -1,44 +1,34 @@
 'use client';
+import pb from "@/app/lib/pocketbase";
+import { useState } from "react";
+import { useRouter } from "next/navigation";
 
-import pb from "@/app/lib/pocketbase"
-import {useState} from "react";
-import {useRouter} from "next/navigation";
-
-
-
-
-export default function CreateNote(){
-    const [title,setTitle,] = useState("");
+export default function CreateNote() {
+    const [title, setTitle] = useState("");
     const [content, setContent] = useState("");
-
     const router = useRouter();
 
-    const create = async()=> {
-        // await fetch('http://127.0.0.1:8090/api/collections/notes/records', {
-        //     method: 'POST',
-        //     headers: {
-        //         'Content-Type': 'application/json',
-        //     },
-        //     body: JSON.stringify({
-        //         title,
-        //         content,
-        //     })
-        // });
-        // setTitle('');
-        // setContent('');
-        // router.refresh();
-        await pb.collection('notes').create({"title":title,"content":content});
+    const create = async () => {
+        // Check if the user is already logged in from localStorage
+        const storedUser = localStorage.getItem('user');
+        if (!storedUser) {
+            alert('User not logged in');
+            return;
+        }
+        await pb.collection('notes').create({ "title": title, "content": content, "user": storedUser });
```

Comment:
usually we write separate services for actions that interact with the BE, rarely do we create them especially in client components, I suggest having a service abstract layer that interacts with pocket base ( maybe a singleton class ) that than exposes various methods like:

create note, to which you pass the validated content from the client to send to the server via this 

also, try catch this, you never know when an error occurs